packages/base-map/src/index.tsx

```diff
@@ -161,6 +161,7 @@ const BaseMap = ({
       }}
       onTouchCancel={clearLongPressTimer}
       onTouchEnd={clearLongPressTimer}
+      preserveDrawingBuffer
```

Comment:
FYI: This prop is no longer supported in maplibre-gl 5.x: https://github.com/maplibre/maplibre-gl-js/releases/tag/v5.0.0. If this feature is important, that change to maplibre will limit our upgrade options.

Reply:
The current `preserveDrawingBuffer` prop touches the same attribute in `MapOptions`, however that attribute is elsewhere in maplibre-gl v5.x, and react-map-gl v7.1 and above removed support for `preserveDrawingBuffer` without adding an alternate way to touch that attribute in maplibre-gl.